src/engines/html5/html5.js

```diff
@@ -392,6 +393,17 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     this._addCueChangeListener();
   }
 
+  /**
+   * Select a new image track.
+   * @param {ImageTrack} imageTrack - The image track object to set.
+   * @returns {void}
+   */
+  selectImageTrack(imageTrack: ImageTrack): void {
+    if (this._mediaSourceAdapter) {
+      this._mediaSourceAdapter.selectImageTrack(imageTrack);
```

Comment:
need to add to base-media-source-adapter to not crash on hls or native

Reply:
it's there
https://github.com/kaltura/playkit-js/blob/FEC-10941/src/engines/html5/media-source/base-media-source-adapter.js#L177